routers/events.router.js

```diff
@@ -0,0 +1,84 @@
+const express = require("express");
+const router = express.Router();
+
+const { Event } = require("../models/event.model");
+
+// GET /api/events/
+router.get("/", (req, res) => {
+  const { query } = req;
+
+  Event.find(query.checkInReady === "true" ? query : undefined)
```

Comment:
I don't feel like this is a protected endpoint. We are allowing the end user to put in any query if they have entered checkInReady === true. I think that we can add some better validation logic on this using middleware, rather than checking on this value here and passing in anything on the DB call if not.

Reply:
@nickbeaird Protecting endpoints is a good question but it's better to address it to the legacy codebase. Please create a task for fixing this API issue and further client cleanup with a clear description of what should be done.
@Jonathanko52 skip this comment.